models/_sector/dex/trades/optimism/dex_optimism_base_trades.sql

```diff
@@ -11,6 +11,7 @@
     , ref('mummy_finance_optimism_base_trades')
     , ref('sushiswap_v1_optimism_base_trades')
     , ref('sushiswap_v2_optimism_base_trades')
+    , ref('firebird_finance_optimism_base_trades')
```

Comment:
If `firebird_finance_optimism_base_trades` is under `dex_aggregator.trades`, then we can remove it from `dex.base_trades`. (As currently dex.base_trades should only contain trades in dex.trades)